config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "fabien@12345",
```

Comment:
The values found in this file should not be hard-coded like this, they should come from the environment variables because this is secret data that should never be exposed to the public like this.

Reply:
Done